src/CarbonAware.WebApi/src/Models/EmissionsForecastBaseDTO.cs

```diff
@@ -13,14 +13,14 @@ public record EmissionsForecastBaseDTO
     /// </summary>
     /// <example>2022-06-01T00:03:30Z</example>
     [JsonPropertyName("requestedAt")]
-    [Required()]
-    public DateTimeOffset RequestedAt { get; set; } = DateTimeOffset.UtcNow;
+    [Required]
+    public DateTimeOffset? RequestedAt { get; set; }
```

Comment:
double-check w/ @bderusha , since I think the default value was required+expected by for the `forecast/current` path, whereas the `forecast/batch` is supposed to populate it

Reply:
Default value introduces mis-behavior/problems since the .NET framework does validation at the controller level. Having default values w/ need, seems like a bad option here.